pkg/metadata/inventories/README.md

```diff
@@ -142,6 +142,7 @@ The payload is a JSON dict with the following fields
     - `DMI`: The Agent successfully used DMI information to fetch the instance ID (only work on EC2 Nitro instances).
     - `UUID`: The hypervisor or product UUID has the EC2 prefix. The Agent knows it's running on EC2 but don't know on
       which instance.
+  - `account_id` - **string**: the AWS account id associated to this host
```

Comment:
We need to think about this feature not as an AWS thing but as a cloud provider generic feature.

What about: 
```suggestion
  - `cloud_provider_account_id` - **string**: The account/subscription ID from the cloud provider.
    - On `AWS` this is the account id associated to this host.
```